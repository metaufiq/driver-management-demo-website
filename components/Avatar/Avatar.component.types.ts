import { User } from "../../index.types"

export type Props = {
  src: string,
  height?: string,
  width?: string
}