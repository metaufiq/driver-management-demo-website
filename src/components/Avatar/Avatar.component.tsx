import styled from "styled-components";
import { Props } from "./Avatar.component.types";

export const Avatar = styled.img<Props>`
  border-radius: 50%;
  padding: 1rem;
  max-height: ${({height})=> height ?? undefined};
  max-width: ${({width})=> width ?? undefined};
`