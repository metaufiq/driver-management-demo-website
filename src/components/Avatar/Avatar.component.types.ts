export type Props = {
  src: string,
  height?: string,
  width?: string
}