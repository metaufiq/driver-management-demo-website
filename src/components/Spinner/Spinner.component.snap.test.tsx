import { assertSnapshots } from "@dmd/test";
import Avatar from ".";
import { Configs } from "@dmd/test/index.types";
import { Props } from "./Spinner.component.types";

describe('SpinnerSnapshot', () => {

  const configs: Configs<Props> = [
    {
      desc: 'should render normal Spinner',
      props:{
        size: '50px'
      }
    }
  ];

  assertSnapshots(Avatar, configs);
});
