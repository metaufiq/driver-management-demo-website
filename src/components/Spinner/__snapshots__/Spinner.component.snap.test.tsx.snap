// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerSnapshot should render normal Spinner 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ iVnQFX"
  />
</DocumentFragment>
`;
